test-common/src/main/java/com/mentoring/database/DbSelects.java

```diff
@@ -16,28 +16,69 @@
             "WHERE u.email = ?";
     private static final String CHARACTER_ID_BY_CHARACTER_NAME = "SELECT character_id FROM skyxp_character " +
             "WHERE character_name = ?";
```

Comment:
This should be changed to:
"SELECT character_id FROM skyxp_character WHERE character_name = ? AND character_id = ?"

Reply:
You mean: "... AND user_id = ?"

Reply:
Modified.

Reply:
Yes, userId :)